go/test/endtoend/backup/transform/backup_transform_utils.go

```diff
@@ -368,7 +368,8 @@ func verifySemiSyncStatus(t *testing.T, vttablet *cluster.Vttablet, expectedStat
 // verifyInitialReplication generates a record on the primary and verifies that the record
 // exists on all tablets in the shard. We also check the PRIMARY as with lossless semi-sync
 // it's possible that a GTID is applied on a replica but not (yet) on the source:
-//   http://my-replication-life.blogspot.com/2013/09/loss-less-semi-synchronous-replication.html
+//
```

Comment:
This is a `gofmt` change

Reply:
hmm... we don't want to drop the link

Reply:
I feel like you should be able to revert the change to this file and `framework_test.go` and still pass CI

Reply:
I'll try reverting. Just FYI I am getting this when resetting to main:
```
➜  vitess git:(frances/shard-actions) git checkout origin/main go/test/endtoend/backup/transform/backup_transform_utils.go
Updated 1 path from 23dd6f55f9
➜  vitess git:(frances/shard-actions) ✗ git status
On branch frances/shard-actions
Changes to be committed:
  (use "git restore --staged <file>..." to unstage)
	modified:   go/test/endtoend/backup/transform/backup_transform_utils.go

➜  vitess git:(frances/shard-actions) ✗ git add .
➜  vitess git:(frances/shard-actions) ✗ git commit -m "Undo changes here"
2022/09/28 13:53:38 4 files OK
Linting go/test/endtoend/backup/transform
WARN [runner] The linter 'structcheck' is deprecated (since v1.49.0) due to: The owner seems to have abandoned the linter.  Replaced by unused.
WARN [runner] The linter 'varcheck' is deprecated (since v1.49.0) due to: The owner seems to have abandoned the linter.  Replaced by unused.
WARN [runner] The linter 'deadcode' is deprecated (since v1.49.0) due to: The owner seems to have abandoned the linter.  Replaced by unused.
WARN [linters context] structcheck is disabled because of generics. You can track the evolution of the generics support by following the https://github.com/golangci/golangci-lint/issues/2649.
go/test/endtoend/backup/transform/backup_transform_utils.go:371: File is not `gofmt`-ed with `-s` (gofmt)
//   http://my-replication-life.blogspot.com/2013/09/loss-less-semi-synchronous-replication.html
```

Reply:
Looking at the gofmt change, the link isn't being dropped actually! It's being moved to the next line.

Reply:
> @deepthi I've renamed it to "External Failover":
> <img alt="Screen Shot 2022-09-28 at 1 06 34 PM" width="1301" src="https://user-images.githubusercontent.com/31225471/192882039-9c67ef03-b309-46e3-9c2b-c1a9ee3fa741.png">

The text under `Tablet` needs to change
`shard master` -> `shard primary`

Reply:
Just updated!